protostar/commands/test/test_command.py

```diff
@@ -120,11 +122,25 @@ def arguments(self) -> List[Command.Argument]:
                 description="Print slowest tests at the end.",
                 default=0,
             ),
+            Command.Argument(
+                name="last-failed",
```

Comment:
todo: consider also using `--lf` like in pytest

Reply:
(`Command.Argument` accepts `short_name`)

Reply:
[use short name](https://github.com/software-mansion/protostar/pull/895/commits/6577565e63905cf1afe03ef81fbd1b7994dba622)